elasticapm/metrics/sets/cpu_linux.py

```diff
@@ -70,7 +70,9 @@ def collect(self):
             delta = {k: new[k] - prev[k] for k in new.keys()}
             cpu_usage_ratio = delta["cpu_usage"] / delta["cpu_total"]
             self.gauge("system.cpu.total.norm.pct").val = cpu_usage_ratio
-            self.gauge("system.memory.actual.free").val = new["mem_available"]
+            # mem_available not present in linux before kernel 3.14 - fallback to mem_free if not present - see #500
```

Comment:
At least some of the other agents are are using `Free + Buffers + Cached` as the result, when `MemAvailable` is not present. Would you please change this to do the same for consistency?

For example, this is the relevant code for the Go agent: https://github.com/elastic/go-sysinfo/blob/master/providers/linux/memory_linux.go#L71

Reply:
Thanks for the pointer, will do.